File: Python/FizzBuzz.py
import time

#defining variables
length_of_round = 00

#replace values divideable by +
fizz = 0  
buzz = 0
fizzbuzz = 0

# length_of_round = int(input("\nEnter length of round desired: "))
# print("\nNote: FizzBuzz = Fizz * Buzz")
# fizz = int(input("Enter divider to replace with 'Fizz': "))
# buzz = int(input("Enter divider to replace with 'Buzz': "))


length_of_round = 100
fizz = 3
buzz = 5
fizzbuzz = fizz * buzz

start = time.time()

def run_round(i):
    global fizz
    global buzz
    global fizzbuzz
    
    output = ""

    if (i % fizzbuzz) == 0:
        output = "FizzBuzz"
    elif (i % fizz ) == 0:
        output = "Fizz"
    elif (i % buzz ) == 0:
        output = "Buzz"
    else:
        output = str(i)

    return output


for i in range(length_of_round):
    print(run_round(i+1))


end = time.time()

print("\nHow long program ran(seconds):")
print(end - start)

File: Node JS/FizzBuzz.js
var length_of_round = 100;
var fizz_divider = 3;
var buzz_divider = 5;
var fizzbuzz_divider = fizz_divider*buzz_divider;

for (var i = 1; i < length_of_round; i++){
    if (i % fizzbuzz_divider == 0){
        console.log("FizzBuzz");
    } else if (i % fizz_divider == 0){
        console.log("Fizz");
    } else if ( i % buzz_divider == 0){
        console.log("Buzz");
    } else{
        console.log(i);
    }
}